### DISCUSSION TITLE
App should be allowed to install without digital signature

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/maui#20351] [Polls]
Published MAUI App should be allowed to install on the users' PC without digitally signed or having certificate. The way it had always worked by downloading a software and running it.

**Comment by mikkpaasuke** [comment]
Eh? AFAIK, code-signing applications so they install without warnings etc on Windows is a Windows-related topic, it's not .NET MAUI specific. I've got a Windows forms application that's been developed for many years, and it's always been necessary to have it code-signed for quite some time.

**Reply by Xyncgas** [reply]
It's relevant because MAUI.Blazor publishing on windows requires certificate and can only publish as app package in Visual Studio or running ```dotnet package``` unlike how you can publish Console programs as exe

There're softwares working and allowing to be downloaded to Windows without a certificate
